roadservice/core/models.py

```diff
@@ -233,13 +293,89 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.type = Role.Type.COUNTY_MODERATOR
 
-    # TODO
-    def assign_county_expert(self):
-        pass
+    @property
+    def county(self):
+        return self.region.get_concrete()
 
     def get_concrete(self):
         return self
 
+    def add_service_team(self, speciality, members_users):
+        for user in members_users:
+            if user.has_role():
+                return None
+        new_team = ServiceTeam.objects.create(county=self.county, speciality=speciality)
+        for user in members_users:
+            Serviceman.objects.create(team=new_team, user=user)
```

Comment:
User should get refreshed from DB (o.w. its role will be a `Serviceman` object).